### PR TITLE
Fixed some Display Abilities

### PR DESCRIPTION
**Pull request by Uveso**
While working on the new automatic-ability function, i found wrong Abilities inside some units.

### CODE REVIEW

File: BlackOpsUnleashed/units/BAA0401/BAA0401_unit.bp [hunk 1]
```diff
@@ -100,9 +100,6 @@ UnitBlueprint {
     },
     Description = '<LOC baa0401_desc>Experimental Satellite',
     Display = {
-        Abilities = {
-            'Orbital',
-        },
```
Comment (aeoncleanse) [inline BlackOpsUnleashed/units/BAA0401/BAA0401_unit.bp:105]: Is this not orbital?

File: BlackOpsUnleashed/units/BAB1106/BAB1106_unit.bp [hunk 2]
```diff
@@ -53,9 +53,6 @@ UnitBlueprint {
     },
     Description = '<LOC bab1106_desc>Mass & Energy Storage',
     Display = {
-        Abilities = {
-            'Storage',
-        },
```
Comment (aeoncleanse) [inline BlackOpsUnleashed/units/BAB1106/BAB1106_unit.bp:58]: Storage can probably storage?

**Comment by aeoncleanse** [comment]
Looks like there could be some issues here. Highlighted a couple. I think all abilities _sshould_ be using the <LOC ... syntax mind, so we get proper translations.

**Comment by Uveso** [comment]
Yes i added the `<LOC ability_xyz>` to all abilities, so we can translate them.
These are all abilities from BlackOps who need translation:

```
ability_groundattack="ability_groundattack"
ability_anti="Anti-Teleport Field:Prevents Teleportation"
ability_gat="Gattling Pulse Laser"
ability_torpturn="Torpedo Redirection System"
ability_shieldisrupt="Shield Disruptor"
ability_antiteleport="Anti-Teleport Field: Prevents Teleportation"
ability_ran="Random Weapon:"
ability_directfire="ability_directfire"
ability_attkdrones="Anti-Air Drones"
ability_energy="High Energy Consumption"
ability_quanRes="Quantum Power"
ability_tar="Forced Targeting"
ability_indirectfire="ability_indirectfire"
ability_lambdafield="Lambda Field"
ability_cloakfield="Cloak Field"
ability_flm="Flamethrower"
ability_teleport="Teleport"
ability_mine="Deploys Land Mines"
ability_lsr="Beam Laser"
ability_rkt="Rocket-propelled Grenades"
```

I deleted the orbital and storage abilitiy because the tooltip already has this information.
Its like engineers with repair or reclaim.

Will be clear if you see the tooltips:
toolabi1
toolabi2

File: BlackOpsUnleashed/units/BEL0308/BEL0308_unit.bp [hunk 3]
```diff
@@ -64,7 +64,7 @@ UnitBlueprint {
     Description = '<LOC bel0308_desc>Artillery Support Bot',
     Display = {
         Abilities = {
-            'Directfire',
+            'ability_directfire',
```
Comment (aeoncleanse) [inline BlackOpsUnleashed/units/BEL0308/BEL0308_unit.bp:67]: Not using the LOC. Just one example.

**Comment by aeoncleanse** [comment]
The abilities display is also shown in other places where that full explanation window isn't visible. It's probably best to keep them. If that means it gets said twice, remove it from the other location.

**Comment by Uveso** [comment]
closed,
next attempt can be found here :  #153

**Pull request closed without merging**